libs/language-server/src/lib/validation/checks/block-definition.ts

```diff
@@ -81,9 +61,18 @@ function checkPipesOfBlock(
       isCompositeBlocktypeDefinition(block.$container) &&
       block.$container.blocks.at(0)?.name === block.name;
 
+    const isExtractorBlock = !blockType.hasInput() && whatToCheck === 'input';
```

Comment:
Naming is somewhat confusing since it may be an ExtractorBlock even if we don't check for it.

Reply:
Will be refactored in a future PR.